### PR TITLE
sys-15 lock engine versions

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
   "main": "src/index.ts",
   "author": "Agriculture Capstone Team",
   "license": "UNLICENSED",
+  "engines": {
```
Comment (beaesguerra) [inline package.json:8]: I think all of these should be using '>=' instead of '=' so that any version greater than specified should be ok

**Review by beaesguerra — CHANGES_REQUESTED** [review]
Please apply the same changes to the other pull requests you had for locking the engine versions

File: package.json [hunk 2]
```diff
@@ -5,6 +5,11 @@
...
+    "node": "= 8.9.1",
+    "npm": "= 5.5.1",
+    "yarn": "= 1.3.2"
```
Comment (Parasrah) [inline package.json]: When we started development these were already specified in the agriboard. For the sake of not breaking peoples development environments I think the scope of this issue should be to sync the package.json engines to the ones that were originally defined at the beginning of the project
```
{
    "node": ">= 8.9.0",
    "npm": ">= 5.5.1",
    "yarn": ">= 1.3.0"
}
```

**Review by Parasrah — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ExtraGravity) [reply package.json:8]: I disagree, I think t o sync our development versions we should be using `=`. Any upgrade should be a conscious one.

**Review by beaesguerra — APPROVED** [review]

**Comment by ExtraGravity** [comment]
@Right2Drive please approve

**Review by Parasrah — APPROVED** [review]

**Pull request merged**